samples/net/wpanusb/sample.yaml

```diff
@@ -6,3 +6,13 @@ tests:
     harness: net
     depends_on: ieee802154 usb_device
     tags: net ieee802154 usb
+  test_wpanusb_frdm_cr20a:
+    harness: net
```

Comment:
Could you add the "harness:" and "tags:" lines under "common:" block, these two tags are now unnecessarily triplicated in this file.

Reply:
This is done now.